_createIndex.js

```diff
@@ -26,7 +26,6 @@ module.exports = function createIndex() {
       }
     },
     mappings: {
-      _doc: {
         // properties
         properties: omitFields({
```

Comment:
nit: do you mind unindenting this over the left?

Reply:
fixed.